ustriage/ustriage.py

```diff
@@ -272,7 +273,7 @@ def parse_dates(start, end=None):
 
 def print_bugs(tasks, open_in_browser=False, shortlinks=True, blacklist=None,
                limit_subscribed=None, oder_by_date=False, is_sorted=False,
-               extended=False):
+               extended=False, filename_save=False, filename_compare=False):
```

Comment:
The default (unset) filenames should be =None rather than =False for proper type consistency.

Reply:
It is Default False and thereby a bool everywhere (consistently), not just here - also in all function definitions, argparse and so on. And a type string if set via the commandline.

Do you mean with type consistency that, since the set value isn't `True` then the unset shouldn't be `False`.
And therefore instead it should be `None` or `String`?
Ok - under that aspect that this is closer I can follow the argument.

Fixed, thanks for the hint!

Reply:
Yep, that's it precisely.